src/go/k8s/helm-chart/charts/redpanda-operator/README.md.gotmpl

```diff
@@ -0,0 +1,24 @@
+# Redpanda Operator
```

Comment:
I am missing some developer documentation, how does this get generated? How will this be updated in the future?

Reply:
[In the commit message](https://github.com/vectorizedio/redpanda/pull/814/commits/a0cfd94d6df896739d005291bc7b3005957edcb7) I put the description how the `README.md` was created.

> operator/helm: Create first helm chart for Redpanda operator
>
> The is scafolding of the helm chart. The `README.md` is generated
> by helm-docs. The `Chart.yaml` is anchor to the 0.1.0 version.

I will create new README for other contributors in `src/go/k8s/helm-chart` directory.